### PR TITLE
Fix problems reported by PHPStan

### PR DESCRIPTION
**Pull request by szepeviktor**
### 📦 Pull Request

@phpstan reported these problems.

@FYJen Please look at the diffs.

### CODE REVIEW

File: lib/HttpClient.php [hunk 1]
```diff
@@ -256,11 +256,11 @@ public function _parse_and_convert_to_api_response($resp_content, $status_code,
 
         throw new \MagicAdmin\Exception\ApiException(
             '',
-            \array_key_exists('status', $resp_content) ? $resp_content->status : null,
+            \property_exists('status', $resp_content) ? $resp_content->status : null,
```
Comment (brianrlewis) [inline lib/HttpClient.php:259]: The arguments passed to property_exists need to be reversed. Example: property_exists($resp_content, 'status')

File: lib/HttpClient.php [hunk 2]
```diff
@@ -256,11 +256,11 @@ public function _parse_and_convert_to_api_response($resp_content, $status_code,
...
             $status_code,
-            \array_key_exists('data', $resp_content) ? $resp_content->data : null,
-            \array_key_exists('message', $resp_content) ? $resp_content->message : null,
-            \array_key_exists('error_code', $resp_content) ? $resp_content->error_code : null,
+            \property_exists('data', $resp_content) ? $resp_content->data : null,
```
Comment (brianrlewis) [inline lib/HttpClient.php:261]: The arguments passed to property_exists need to be reversed.

File: lib/HttpClient.php [hunk 3]
```diff
@@ -256,11 +256,11 @@ public function _parse_and_convert_to_api_response($resp_content, $status_code,
...
+            \property_exists('message', $resp_content) ? $resp_content->message : null,
```
Comment (brianrlewis) [inline lib/HttpClient.php:262]: The arguments passed to property_exists need to be reversed.

File: lib/HttpClient.php [hunk 4]
```diff
@@ -256,11 +256,11 @@ public function _parse_and_convert_to_api_response($resp_content, $status_code,
...
+            \property_exists('error_code', $resp_content) ? $resp_content->error_code : null,
```
Comment (brianrlewis) [inline lib/HttpClient.php:263]: The arguments passed to property_exists need to be reversed.

**Review by brianrlewis — CHANGES_REQUESTED** [review]
Thank you for submitting these fixes. Before this can be merged there is one problem that needs to be fixed and another small tiddying up detail that should also be addressed.

1) The order of the arguments passed to `property_exists` needs to be reversed in `HttpClient.php`. I made in-file comments regarding this.

2) The use statement (`use MagicAdmin\Exception;`) on line 5 of [Token.php](https://github.com/magiclabs/magic-admin-php/blob/0e408e4a910931a94805e618bb6eb8222476c15f/lib/Resource/Token.php) can be removed, just like it was removed from `DidToken.php`.

**Comment by szepeviktor** [comment]
> there is one problem that needs to be fixed and another small tiddying up detail that should also be addressed.

Your wish is my command.

**Comment by brianrlewis** [comment]
Great, thank you.

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request merged**